### DISCUSSION TITLE
Handlebars #if support

### DISCUSSION THREAD

**Discussion by deckster271** [bootstrap-email/bootstrap-email#147] [Ideas]
Hi there,
If I include a handlebars #if statement it gets stripped out of the render.

e.g.

{{#if var}}
etc....
{{/if}}

The render ignores the if statements

Thanks

**Comment by stuyam** [comment]
Can you give me a more comprehensive example? Like a full html document test so I can better test what is happing here. I will look into it though, thanks 👍

**Reply by deckster271** [reply]
Sorry it's taken a while.....
This is the example html I'm working with:
`<div class="stack-row">
            {{#if photo0}}
            <img src={{photo0}} style="height: auto; line-height: 100%; outline: none; text-decoration: none; border: 5px none;" width="100" height="100">
            {{/if}}
            {{#if photo1}}
            <img src={{photo1}} style="height: auto; line-height: 100%; outline: none; text-decoration: none; border: 5px none;" width="100" height="100">
            {{/if}}
            {{#if photo2}}            
            <img src={{photo2}} style="height: auto; line-height: 100%; outline: none; text-decoration: none; border: 5px none;" width="100" height="100">
            {{/if}}
            {{#if photo3}}
            <img src={{photo3}} style="height: auto; line-height: 100%; outline: none; text-decoration: none; border: 5px none;" width="100" height="100">
            {{/if}}
            {{#if photo4}}
            <img src={{photo4}} style="height: auto; line-height: 100%; outline: none; text-decoration: none; border: 5px none;" width="100" height="100">
            {{/if}}
          </div>`

The {{#if}} and {{/if}} are as per Handlebars config:
https://handlebarsjs.com/guide/builtin-helpers.html

When I then render it, the {{#if}} & {{/if}} get removed from the output.

Thanks for looking into it.....
